packages/styled-components/package.json

```diff
@@ -66,7 +66,7 @@
     "@babel/traverse": "^7.4.5",
     "@emotion/is-prop-valid": "^0.8.8",
     "@emotion/unitless": "^0.7.4",
-    "babel-plugin-styled-components": ">= 1",
+    "babel-plugin-styled-components": "^1.12.0",
```

Comment:
I'd prefer to continue using the `>=` notation so we don't have to update the library here if v2 comes out that doesn't contain incompatible changes with styled-components core

Reply:
np, will force push that change